src/extension/baser.ts

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Uri, window, workspace } from 'vscode';
+import '../shared/extension';
+import { Store } from './store';
+
+export class Baser {
+    constructor(
+        private readonly distinctLocalNames: Map<string, string>,
+        private readonly store: Pick<Store, 'distinctArtifactNames'>) {
+    }
+
+    private basesArtifactToLocal = new Map<string, string>([]) // <artifactPath, localPath>
```

Comment:
>[] [](start = 59, length = 2)

Curious as to why an empty array is being passed?

Reply:
It's extra. Now removed.

---
In reply to: [441617576](https://github.com/microsoft/sarif-vscode-extension/pull/282#discussion_r441617576) [](ancestors = 441617576)